### DISCUSSION TITLE
Is apt.releases.teleport.dev being updated for stable/v13 releases?

### DISCUSSION THREAD

**Discussion by MattManjosKodiak** [gravitational/teleport#39310] [Q&A] [answered]
While looking at 

```
https://apt.releases.teleport.dev/ubuntu/dists/jammy/stable/v13/binary-amd64/Packages
```

I noticed that two of the most recent releases, `13.4.17` and the just-released `13.4.18` are missing. Is this intended or will these stable/v13 releases be updated to the mirror soon?

**Comment by camscale** [comment, marked as answer]
These releases should now be present. This issue also affected 14.3.7 which is also now present in the package archives.

A recent change to our release publishing process sent v13 and v14 images to the development repository instead of the production repository. This has been fixed and run to re-publish these three releases (13.4.17, 13.4.18, 14.3.7) to the production repository.

**Comment by MattManjosKodiak** [comment]
Thanks! Looks good now!
